### DISCUSSION TITLE
default: No such process

### DISCUSSION THREAD

**Discussion by alexdeslauriers16** [scivision/PyLivestream#98] [Q&A]
Trying to go live on twitch and I get the error:

Press Enter to go live on ['twitch']    Or Ctrl C to abort.

 ffmpeg -loglevel error -f x11grab -s 640x480 -i :0.0+50,30 -f pulse -i default -codec:v libx264 -pix_fmt yuv420p -preset veryfast -b:v 500k -g 60 -codec:a aac -b:a 96k -ar 44100 -maxrate 500k -bufsize 250k -strict experimental -f flv rtmp://live-ymq.twitch.tv/app/{MY_KEY_WAS_HERE}

default: No such process

My code is simply

import pylivestream.api as pls

pls.stream_screen('pilivestream.ini', 'twitch')

And Im running this on raspberry pi.

**Comment by alexdeslauriers16** [comment]
Im having the same error when trying to stream webcam, which is a bigger issue for me

**Comment by scivision** [comment]
I am not sure, this command works for me to localhost or other sites. It may be something missing like v4l2 from the system or from FFmpeg as it was built.
